include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -276,7 +276,7 @@ void SplineBuilder<
         Solver>::
 operator()(
         ddc::ChunkSpan<double, ddc::DiscreteDomain<bsplines_type>, Layout, MemorySpace> spline,
-        ddc::ChunkSpan<double, interpolation_domain_type, Layout, MemorySpace> vals,
+        ddc::ChunkSpan<double const, interpolation_domain_type, Layout, MemorySpace> vals,
```

Comment:
```suggestion
        ddc::ChunkSpan<double, interpolation_domain_type, Layout, MemorySpace> vals,
```

Reply:
Why do you want to revert that ? Isn't read-only array ?